README.md

```diff
@@ -1,2 +1,70 @@
+
+You can learn more about the Notary Project on the [notaryproject.dev](https://notaryproject.dev) website. 
+
+## Repositories
+
+Here is a list of repositories under the Notary Project organization
+
+| Repository | Description |
+| ----- | -----|
+| [.github](https://github.com/notaryproject/.github) | This repository contains the Notary Project governance and other common documents that are shared across repositories. |
+| [meeting-notes](https://github.com/notaryproject/meeting-notes) | This repository contains the archived meeting notes. |
+| [notary](https://github.com/notaryproject/notatry) | This repository contains the source code for the server and the client of the TUF-based implementation. |
+| [notaryproject](https://github.com/notaryproject/notatryproject) | This repository contains the Notary Project requirements, scenarios,  specifications, and security audits. |
+| [notaryproject.dev](https://github.com/notaryproject/notatryproject.dev) | This repository contains the source code and content for the [Notary Project website](https://notaryproject.dev). | 
+| [notation](https://github.com/notaryproject/notation) | This repository contains the source code for the convenience CLI implementation of the Notary signing and verification flow. |
+| [notation-go](https://github.com/notaryproject/notation-go) | This repository contains the source code for the convenience Golang library implementation of the Notary signing and verification flow. |
+| [notation-core-go](https://github.com/notaryproject/notation-core-go) | This repository contains the source code for the Golang library implementation of the Notary signature specification and wrapping (COSE and JWS). |
+| [roadmap](https://github.com/notaryproject/roadmap) | This repository is intended for keeping track of development activitis in the Notary project. It may be retired in the future. |
+| [tuf](https://github.com/notaryproject/tuf) | This repository is intended for prototyping the storage of TUF metadata in OCI-compliant registries. It is not under active development at the moment but there are plans to revive it in the future. |
+
+## Project Status
+
+The Notary Project is in active development. The latest release announcement are published on the [Notary Project blog](https://notaryproject.dev/blog/). The Notary community uses the [project board](https://github.com/orgs/notaryproject/projects/10) for project planning and status tracking. You can also use GitHub milestones to track the progress of each repository:
```

Comment:
typo:

"The latest release announcement `are` published" ==> "The latest release announcement `is` published"